satpy/readers/ahi_hsd.py

```diff
@@ -1,13 +1,14 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 #
-# Copyright (c) 2014-2018 PyTroll developers
+# Copyright (c) 2014-2019 PyTroll developers
 #
 # Author(s):
 #
 #   Adam.Dybbroe <adam.dybbroe@smhi.se>
 #   Cooke, Michael.C, UK Met Office
 #   Martin Raspaud <martin.raspaud@smhi.se>
+#   Simon R. Proud <simon.proud@physics.ox.ac.uk>
```

Comment:
I'd really like to get away from individual author lists in files. @mraspaud can argue with me, but I'd rather have this list removed than add more to it.